easybuild/easyconfigs/b/Bash/Bash-4.3-GCC-4.9.2.eb

```diff
@@ -0,0 +1,31 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Computer Science and Communications Research Unit
+# Authors::   Valentin Plugaru <valentin.plugaru@gmail.com>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_05-06.html
+##
+
```

Comment:
this is missing `easyblock = 'ConfigureMake'` too

the build works with EasyBuild v1.16.1 (cfr. test report), but it won't with EasyBuild v2.0 (and Jenkins will tell you that with failing tests)